watertap/flowsheets/anaerobic_digester/modified_ADM1_flowsheet.py

```diff
@@ -0,0 +1,348 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2024, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+"""
+Flowsheet example full Water Resource Recovery Facility
+(WRRF; a.k.a., wastewater treatment plant) with ASM2d and ADM1 with P extension.
+
+The flowsheet follows the same formulation as benchmark simulation model no.2 (BSM2)
+but comprises different specifications for default values than BSM2.
+"""
+
+# Some more information about this module
+__author__ = "Chenyu Wang, Adam Atia, Alejandro Garciadiego, Marcus Holly"
+
+import pyomo.environ as pyo
+from pyomo.network import Arc
+
+from idaes.core import (
+    FlowsheetBlock,
+    # UnitModelCostingBlock,
```

Comment:
Do we want to include costing for the AD in this flowsheet? If not, we can delete this